test/e2e/network/ingress.go

```diff
@@ -1112,19 +1112,40 @@ var _ = SIGDescribe("Ingress API", func() {
 
 		// Ingress resource delete operations
 		ginkgo.By("deleting")
+
+		expectFinalizer := func(ing *networkingv1.Ingress) {
```

Comment:
making this a helper method will make line numbers hard to follow when a failure happens... it won't be clear which step failed. suggest passing in a message or something so we can see which check failed

Reply:
Good call, updated